src/transaction.ts

```diff
@@ -448,6 +489,7 @@ export class Transaction implements TransactionStorageStructure {
       !txn.nonParticipation &&
       (txn.voteKey ||
```

Comment:
why is this check needed here. It's checking one of the fields is set?

Reply:
This is to make sure if any of the online keyreg fields are provided, then all must be provided